#!/usr/bin/env python

PACKAGE = "first_project"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("wheel_odom", str_t,      2, "A string parameter", "wheel_odom")
gen.add("gps_odom", str_t,        3, "A string parameter", "gps_odom")

exit(gen.generate(PACKAGE, "lidar_remap", "parameters"))